src/content/index.tsx

```
@@ -2,14 +2,14 @@ import { v4 as uuidv4 } from "uuid";
 import "./index.less";
 
-const mainGinkgo = () => {
+const mainGinkgoo = () => {
   let port: chrome.runtime.Port | null = null;
 
   const handleMessage = (event: MessageEvent) => {
     const message = event.data;
     const { type, ...otherInfo } = message;
-    const [_, source, target] = type.split("-");
-    // console.log("[Ginkgo] ContentScript handleMessage", event, type, type.startsWith("ginkgo-page-"));
+    const [_, source, target] = type?.split("-");
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)